File: java/Account.java

class Account {
    Integer id;
    String name;
    Integer document;
    Integer fone;
    String email;
    String password;

    //Metodo
    public  Account(Integer document,String email,String password) {
        this.document = document;
        this.email = email;
        this.password = password;  
    }
}

File: php/account.php
<?php
class Account {
    public $id;   
    public $names;   
    public $document;  
    public $email;  
    public $password;   

    public function __construct($names,$document,$email,$password)
    {
      
        $this->names = $names;
        $this->document = $document;
        $this->email = $email;
        $this->password = $password;
    }
} 